addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13574,7 +13574,17 @@ msgctxt "#21342"
 msgid "There are currently no updates available for this add-on."
 msgstr ""
 
-#empty strings from id 21343 to 21357
+#: system/settings.xml
+msgctxt "#21343"
+msgid "Use video tags"
+msgstr ""
+
+#: system/settings.xml
+msgctxt "#21344"
+msgid "If mp4 or mkv files have tags, use this to for library metadata"
```

Comment:
to -> too ?

Reply:
Ehrm 'to fill' or 'for' and i ended up with useless hybrid.

Reply:
hehe

Reply:
removed to 'to'